crml/staking/src/lib.rs

```diff
@@ -378,20 +378,25 @@ pub struct UnlockChunk<Balance: HasCompact> {
 pub struct StakingLedger<AccountId, Balance: HasCompact> {
 	/// The stash account whose balance is actually locked and at stake.
 	pub stash: AccountId,
-	/// The total amount of the stash's balance that we are currently accounting for.
-	/// It's just `active` plus all the `unlocking` balances.
+	/// The total amount of the stash's balance (`active` plus any `unlocking` balances).
 	#[codec(compact)]
 	pub total: Balance,
-	/// The total amount of the stash's balance that will be at stake in any forthcoming
-	/// rounds.
+	/// The amount of the stash's balance that will be at stake in any forthcoming
+	/// eras. i.e it will affect voting power in the next election.
+	/// It could lessen in the current round after unbonding.
 	#[codec(compact)]
 	pub active: Balance,
-	/// Any balance that is becoming free, which may eventually be transferred out
+	/// Any balance that has been unbonded and becoming free, which may eventually be transferred out
 	/// of the stash (assuming it doesn't get slashed first).
 	pub unlocking: Vec<UnlockChunk<Balance>>,
 }
 
 impl<AccountId, Balance: HasCompact + Copy + Saturating + AtLeast32Bit> StakingLedger<AccountId, Balance> {
+	/// The amount of stash's funds slashable as of right now.
+	/// It should remain slashable until the bonding duration expires and it is withdrawn.
+	fn slashable_balance(&self) -> Balance {
```

Comment:
As this is not a public getter, it seems adding doc comments to "self.total" and or renaming it could do the job and could be better than a function call.

Reply:
makes sense to me. the diff will be quite big so will address in a follow up PR